packages/esm-service-queues-app/src/queue-table/queue-table.test.tsx

```diff
@@ -67,9 +67,12 @@ const defaultProps = {
 };
 
 describe('QueueTable', () => {
+  let consoleSpy: jest.SpyInstance;
+
   beforeEach(() => {
     mockUseSession.mockReturnValue(mockSession.data);
     mockUseConfig.mockReturnValue(configDefaults);
+    consoleSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
This hides the [no visit queue number attribute configured](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-service-queues-app/src/queue-table/cells/queue-table-visit-attribute-queue-number-cell.component.tsx#L12) console error from the logs.